### ISSUE TITLE
If something's wrong with equipment, game may falsely report that the size of an equipment is 0

### ISSUE THREAD

**Issue by ncrecc** [TerryCavanagh/diceydungeons.com#1099]
![](https://i.frogbox.es/lqp)
(What's wrong with the equipment in question? I don't know - the error isn't very helpful. Perhaps it encountered an error earlier on and failed to parse the size of Trinity Charm_upgraded. Trinity Charm and its upgraded version are intentionally the same here for debugging purposes)

**Comment by ncrecc** [comment]
This was about commas, duhh. Would still be nice if game said something like "size provided is not a number!" rather than "size is 0" - maybe even explicitly say something like "size provided is not a number! did you use commas instead of bracketed semicolons?"

**Comment by TerryCavanagh** [comment]
Yep! I'd like to improve this particular error message, since it's the one people tend to get if they make a mistake with commas in the spreadsheets. Unfortunately it's kinda difficult to detect...

**Comment by TerryCavanagh** [comment]
Oh, I guess it could just tell you what value was passed incorrectly: "Size for Trinity Charm must be either 1 or 2 - " else: return last dice[gray](Reuseable)" is not a valid value" would be a much more useful error message, right?
